### PR TITLE
Fix php doc so that hasAnnotation works properly

### PR DESCRIPTION
**Pull request by sharidas**
An asterisk was missing in the php doc which was
causing problem when user is enabled/disabled
from UI.

Signed-off-by: Sujith H <sharidasan@owncloud.com>

## Description

The call to `setEnabled()` was not happening because of missing asterisk ('*') in the phpdoc. And hence `NotAdminException()` was triggered here: https://github.com/owncloud/core/blob/stable10/lib/private/AppFramework/Middleware/Security/SecurityMiddleware.php#L132, resulting in error.

## Related Issue

https://github.com/owncloud/core/issues/31280

## Motivation and Context

The call to `setEnabled()` was not happening because of missing asterisk ('*') in the phpdoc. And hence `NotAdminException()` was triggered here: https://github.com/owncloud/core/blob/stable10/lib/private/AppFramework/Middleware/Security/SecurityMiddleware.php#L132, resulting in error.

## How Has This Been Tested?

- [x] Followed the steps mentioned in the issue.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: settings/Controller/UsersController.php [hunk 1]
```diff
@@ -869,7 +869,7 @@ public function changeMail($token, $userId) {
 		return new RedirectResponse($this->urlGenerator->linkToRoute('settings.SettingsPage.getPersonal', ['changestatus' => 'success', 'user' => $userId]));
   }
   
-  /*
+  /**
```
Comment (phil-davis) [inline settings/Controller/UsersController.php]: While we are here, the indent of line 870 and this line 872 can be fixed.

**Comment by phil-davis** [comment]
Note: needs the same fix in:
https://github.com/owncloud/user_management/blob/master/lib/Controller/UsersController.php#L870
where this code happens in the separate ``user_management`` app for ``core`` ``master`.

**Comment by sharidas** [comment]
@phil-davis Yes I will create the PR for user-management. But the failure which was troubling enable/disable by subadmin doesn't affect there.

**Comment by phil-davis** [comment]
At line 787 there is also missing indent. It would be good if you can quickly review the indent layout of this file and see if there is anything else that could be tidied.

**Comment by sharidas** [comment]
Created PR in user_management : https://github.com/owncloud/user_management/pull/30

**Review by phil-davis — APPROVED** [review]
Works, and the indent looks neater also.

**Comment by phil-davis** [comment]
Luv that ``codecov`` - a little change like this takes 16% off the project-wide code coverage!

**Pull request merged**